lib/tasks/bundlers/generateStandaloneAppBundle.js

```diff
@@ -76,12 +77,18 @@ module.exports = async function({workspace, dependencies, taskUtil, options: {pr
 			`unable to generate complete bundles for such projects.`);
 	}
 
-	// If an application does not have a namespace, its resources are located at the root. Otherwise in /resources
```

Comment:
By removing this behavior, we need to make sure to fail in case we don't have a namespace (as already planned for 3.0)

Reply:
Yes, correct!